README.md

```diff
@@ -139,12 +143,12 @@ This is how the method behaved before version 1.19.0.
 ### `getTokenSilentlyVerbose`
 
 Original `getTokenSilently` method with `detailedResponse` forced to `true`.
-Splitting the method was required to support proper types, as wrapping a function and trying to infer its return type won't work if the original function has an overloaded signature.
+Splitting the original method was required to support proper types, as wrapping a function and trying to infer its return type won't work if the original function has an overloaded signature.
 
 ### `initializationCompleted`
 
 Returns a Promise resolving when initialization completes.
-Use this when you need to be sure initialization completed and cannot use `onInitializationCompleted` hook.
+Use this when you need to be sure initialization is completed and you cannot use `onInitializationCompleted` hook.
```

Comment:
AFAIK just `completed` here is a legit usage, but I may be wrong
Is there a reason for the change or is it just out of how it sounded to you?

Reply:
I'm sure about the `you` since there is no specified subject in the phrase, and either we needed to add the `you` or at least change the phrase just a little bit.
About the `is completed`, yes it is more how it sounded to me. I think that the contraction there was a little bit too much and needed an explicit verb. But I do not have a rule telling me that we must place here the is. I would need to check a proper grammar book.